### ISSUE TITLE
@next.2 - Import types in d.ts files contain relative paths

### ISSUE THREAD

**Issue by bendersej** [fuse-box/fuse-box#1339]
```
fuse-box/core/FuseBox.d.ts
```
```
process(bundleData: BundleData, bundleReady?: () => any): Promise<import("../../../../../../../Users/nc/work/fuse-box/fuse-box/src/Utils").Concat>;
addShims(): void;
initiateBundle(str: string, bundleReady?: any): Promise<import("../../../../../../../Users/nc/work/fuse-box/fuse-box/src/Utils").Concat>;
```

Most likely related to: https://github.com/Microsoft/TypeScript/issues/24599.

**Cause?:** Local TS version used to build `fuse-box` differs from the one in `package.json` (hasn't changed for the last 6 months). Either global or a result of the work on this [branch](https://github.com/fuse-box/fuse-box/pull/1331).

Keep up the good work!

**Comment by nchanged** [comment]
@bendersej thanks! can you please verify it on `3.5.0-next.3`

**Comment by bendersej** [comment]
@nchanged Fixed with `3.5.0-next.3`, thanks! :)
